### ISSUE TITLE
Custom content and container attributes are not optional!

### ISSUE THREAD

**Issue by nrandell** [Mastermindzh/react-cookie-consent#141]
Easy thing to fix, but I need to add the following in order to use it

```
        customContainerAttributes={{}}
        customContentAttributes={{}}

```

**Comment by Mastermindzh** [comment]
Ah yes... that is a mistake I made yesterday :)

I will solve it tonight!

**Comment by Mastermindzh** [comment]
Solved in version [7.1.1](https://github.com/Mastermindzh/react-cookie-consent/releases/tag/7.1.1)
